monitoring/accessing-third-party-monitoring-uis-and-apis.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="accessing-third-party-monitoring-uis-and-apis"]
+= Accessing third-party monitoring UIs and APIs
+include::modules/common-attributes.adoc[]
+:context: accessing-third-party-monitoring-uis-and-apis
+
+toc::[]
+
+[role="_abstract"]
+In {product-title} {product-version}, you cannot access third-party web browser user interfaces (UIs) for the following monitoring components: Alertmanager, Thanos Ruler, and Thanos Querier. 
+However, you can access web UIs for Grafana and Prometheus, although this access is deprecated and is planned to be removed in a future {product-title} release.
+In addition, you can access web service APIs for third-party monitoring components from the command line interface (CLI).
+
+// Accessing web UIs for third-party monitoring components
+include::modules/monitoring-accessing-third-party-monitoring-uis.adoc[leveloffset=+1]
+
+// Accessing service APIs for third-party monitoring components
+include::modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
If these are additional resources for the assembly, not the preceding module, apply the following formatting instead.
```suggestion
== Additional resources
```
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
Fixed, and added the attributes for Additional Resources sections per the guidance.